replica/database.hh

```diff
@@ -895,7 +895,7 @@ public:
      * Return a set of the sstables names that contain the given
      * partition key in nodetool format
      */
-    future<std::unordered_set<sstring>> get_sstables_by_partition_key(const sstring& key) const;
+    future<std::unordered_set<sstables::shared_sstable>> get_sstables_by_partition_key(const sstring& key) const;
```

Comment:
why do we need to return an `unordered_set<>` instead of a plain `vector<>` ?

Reply:
Because it currently does so. Presumably the amount of sstables here can be large enough to consider vector size to be large enough to care

Reply:
okay, note to myself. it could be a chance to change it to chunked_vector.